.buildkite/pipeline.yml

```
@@ -136,6 +136,50 @@ steps:
 
   # ----------
 
+  # Run CVE checks for the main branch
+  - group: cve image scans
+    steps:
+
+      # scan the operator image for CVEs on merge to main or nightly builds
+      - label: ":buildkite: cve-checks"
+        #if: build.branch == "main"
+        depends_on:
+          - "operator-image-build"
+        command: |
+          export CONTAINER=$$(buildkite-agent meta-data get operator-image)
+          cat <<- YAML | buildkite-agent pipeline upload
+          steps:
+            - label: "Trigger cve-slo-status pipeline for $$CONTAINER"
+              trigger: cve-slo-status
+              build:
+                env:
+                  CONTAINER: "$$CONTAINER"
+              soft_fail: true
+          YAML
+        agents:
+          image: docker.elastic.co/ci-agent-images/cloud-k8s-operator/buildkite-agent:e808dbe5
+          memory: "128Mi"
```

Surprisingly, 128Mi doesn't seem to be enough to clone the repo. Let's double?
```suggestion
          memory: "256Mi"
```